state/factory/workingset.go

```diff
@@ -339,200 +170,3 @@ func (ws *workingSet) PutState(pkHash hash.PKHash, s interface{}) error {
 	}
 	return ws.accountTrie.Upsert(pkHash[:], ss)
 }
-
-// clearCache removes all local changes after committing to trie
-func (ws *workingSet) clearCache() {
```

Comment:
i added Snapshot() and Revert(), so still need this to clear the snapshots, you can see the addition when you merge to latest

or you can delete this func, and add the addition about snapshots right after calling Commit() in RunAction()

Reply:
also rename it to clear() if you keep the func